docs/writing-docs/build-documentation.md

```diff
@@ -50,7 +50,7 @@ You can use any hosting provider to deploy your documentation, for instance:
 - [Netlify](https://www.netlify.com/)
 - [S3](https://aws.amazon.com/en/s3/)
 
-#### Learn more about Storybook documentation
+**Learn more about Storybook documentation**
```

Comment:
@kylegach this is a nice suggestion. I have a couple of items in the works and will apply this pattern as well.